Dockerfile

```diff
@@ -1,10 +1,46 @@
-FROM node:5
+FROM node:boron
```

Comment:
Worth noting that Michael may have already read / written the code in the related PR.

(just trying to keep info in sync across repos!)